### PR TITLE
Added possibility to enable/disable satalite feature 

### PR DESCRIPTION
**Pull request by galadril**
Added possibility to enable/disable satalite feature

### CODE REVIEW

**Comment by alorma** [comment]
:+1:

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 1]
```diff
@@ -250,10 +252,15 @@ private void setUpFloatingButtons() {
...
-      btnSatellite.setImageResource(map.getMapType() == MAP_TYPE_SATELLITE ? R.drawable.ic_satellite_off : R.drawable.ic_satellite_on);
-    });
+    if(enableSataliteView) {
+      btnSatellite.setOnClickListener(view -> {
+        map.setMapType(map.getMapType() == MAP_TYPE_SATELLITE ? MAP_TYPE_NORMAL : MAP_TYPE_SATELLITE);
+        btnSatellite.setImageResource(map.getMapType() == MAP_TYPE_SATELLITE ? R.drawable.ic_satellite_off : R.drawable.ic_satellite_on);
+      });
+    }
+    else{
+      btnSatellite.setVisibility(View.GONE);
```
Comment (marcserrascmspain) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: Maybe `btnSatellite.setVisibility(enableSataliteView ? View.VISIBLE : View.GONE);` is enough

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 2]
```diff
@@ -632,6 +639,9 @@ private void getTransitionBundleParams(Bundle transitionBundle) {
     if (transitionBundle.keySet().contains(BACK_PRESSED_RETURN_OK)) {
       shouldReturnOkOnBackPressed = transitionBundle.getBoolean(BACK_PRESSED_RETURN_OK);
     }
+    if (transitionBundle.keySet().contains(ENABLE_SATALITE_VIEW)) {
+        enableSataliteView = transitionBundle.getBoolean(BACK_PRESSED_RETURN_OK);
```
Comment (marcserrascmspain) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: wrong Key, use ENABLE_SATALITE_VIEW

Comment (galadril) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: Woops

**Pull request closed without merging**